File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao11.py
qtd_quadrinhos = int(input("Quantos quadrinhos você possue ? "))
i = 1
total_gasto = 0
while i <= qtd_quadrinhos:
    valor_quadrinho = float(input(f"Digite o valor do {i}º quadrinho: "))
    total_gasto += valor_quadrinho
    i += 1

print(f"O valor total gasto em quadrinhos é de:      R${total_gasto}")

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Kotlin/questao11.kt
fun main(){
    println("Quantos quadrinhos você possue ?")
    var qtd_quadrinhos = readLine()!!.toInt()
    var i = 1
    var total_gasto = 0.0f
    while (i <= qtd_quadrinhos) {
        println("Digite o valor do ${i}º quadrinho: ")
        var valor_quadrinho = readLine()!!.toFloat()
        total_gasto += valor_quadrinho
        i += 1
    }   

    println("O valor total gasto em quadrinhos é de:      R$${total_gasto}")
}